src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java

```diff
@@ -39,7 +46,7 @@ public ResponseEntity<Iterable<User>> getAllUsers() {
         }
     }
 
-    @GetMapping("/user-roles")
+    @RequestMapping(value = { "/api/permissions/user-roles"}, method=RequestMethod.GET)
```

Comment:
The endpoint mapping before is the correct way of doing it (GetMapping). Just FYI, we made a system wide change to that way of doing things a couple weeks ago.